src/app/docs/text-editor/text-editor-docs.component.html

```diff
@@ -14,9 +14,12 @@
       </p>
     </sky-docs-demo-page-summary>
 
-    <!-- <sky-docs-demo>
-      Demo here.
-    </sky-docs-demo> -->
+    <sky-docs-demo>
```

Comment:
Should the demo include a *Show options* dropdown to let users include/exclude the optional menu bar and toolbar options listed at https://developer.blackbaud.com/skyux-text-editor/docs/text-editor#section-options?

FWIW, I don't think this is necessary, but just throwing it out there to make sure we considered it.

Reply:
I was really just going for MVP here since we didn't have any specs to go off of.

Reply:
Makes sense to me.